pynicotine/gtkgui/application.py

```diff
@@ -344,8 +344,8 @@ def on_invalid_password(self):
         from pynicotine.gtkgui.widgets.dialogs import OptionDialog
 
         title = _("Invalid Password")
-        msg = _("User %s already exists, and the password you entered is invalid. Please choose another username "
-                "if this is your first time logging in.") % config.sections["server"]["login"]
+        msg = _("Either user %s already exists, or the password you entered is invalid."
```

Comment:
A password can only be invalid if the user exists.

Reply:
"right" -> "correct"